### DISCUSSION TITLE
Bypass/Turn off Recaptcha for Automated Functional Testing

### DISCUSSION THREAD

**Discussion by harsha-mohan** [django-recaptcha/django-recaptcha#255] [Q&A] [answered]
I currently have this package implemented in my project, and it works fine.

I was wondering how you can turn off this package for automated functional testing? I looked at the example of using test keys for RECAPTCHA_PUBLIC_KEY and RECAPTCHA_PRIVATE_KEY and this tag in settings.py SILENCED_SYSTEM_CHECKS = ['captcha.recaptcha_test_key_error'], but it does not work. I am still presented with this below image, and it does not pass my test case. Am I doing something wrong here?

Any suggestions would be really helpful!

![Screen Shot 2021-03-22 at 11 36 31 PM](https://user-images.githubusercontent.com/26445357/112089306-c355bb00-8b67-11eb-9362-eeddc8abcc88.png)

**Comment by thibaudcolas** [comment, marked as answer]
The workarounds described in #215 and #218 might help?
